recipe/meta.yaml

```diff
@@ -52,14 +53,20 @@ requirements:
 test:
   requires:
     - {{ compiler('c') }}
+  imports:
+    - vapoursynth
   files:
     - test_autoload.sh  # [not win]
     - test_autoload.bat  # [win]
     - crash-plugin.c
+  source_files:
+    - test
   commands:
     - vspipe --version
     - '"$PYTHON" -c "from vapoursynth import core; print(core.version())"'  # [not win]
     - '"%PYTHON%" -c "from vapoursynth import core; print(core.version())"'  # [win]
+    - '"%PYTHON%" -m unittest discover -s test -p "*test.py"'  # [win]
+    - '"$PYTHON" -m unittest discover -s test -p "*test.py"'  # [not win]
```

Comment:
Why do we need to run the unit tests in conda builds? I don't see it in other recipes as the build resource is so precious that [they are not even willing to use git](https://conda-forge.org/docs/maintainer/adding_pkgs/#build-from-tarballs-not-repos).

Reply:
I think it is important to run the unit test, It can prevent producing the brocken packages.

Reply:
Of course, running the unit test is safer in general. However, the unit tests are focused on the program logic, not packaging. Conda-forge is only a distribution project whose computation resource is not meant for helping the software maintainers test their software. Not even for platform-dependent packages like [libhwy](https://github.com/conda-forge/libhwy-feedstock).

If you insist on adding tests, we need more concrete reasons rather than general guidelines. This is beyond my scope of authority and I will have to bring someone from the conda-forge org into the discussion before merging this PR.